### PR TITLE
IRSA-4161_limit_the_default_display_of_the_middle_plane_of_a_cube

### PR DESCRIPTION
**Pull request by deannaji**
Jira: https://jira.ipac.caltech.edu/browse/IRSA-4161
K8s: https://irsa-4161.irsakudev.ipac.caltech.edu/applications/sofia/
ife PR: https://github.com/IPAC-SW/irsa-ife/pull/166

### CODE REVIEW

File: src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java [hunk 1]
```diff
@@ -38,7 +38,11 @@ public FileAnalysisReport analyzeFits(FileAnalysisReport inputReport,
         String code = params.getOrDefault("product_type", "");
         String level = params.getOrDefault("processing_level", "").toUpperCase();
         String inst = params.getOrDefault("instrument", "").toUpperCase();
-        inputReport.setAdditionalImageParams(Collections.singletonMap("CUBE_FIRST_FRAME","50%"));
+        String configuration = params.getOrDefault("configuration", "").toUpperCase();
+
+        if(! inst.equals("FORCAST") && ! configuration.equals("IMAGING")) {
```
Comment (robyww) [inline src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java:43]: I don’t know exactly which the condition is but it seems this is open for error. There must be many things that is not forcast and not imaging.  If this is what you want then ok.

**Review by robyww — APPROVED** [review]
Code looks good as long as the condition is really what you want.

File: src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java (hunk 1 above)
Comment (deannaji) [reply src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java:43]: Ya, for now those two conditions that I need to catch, everything else should go through the "set cube to middle" process. This is how it was before this ticket.

**Comment by ejoliet** [comment]
@deannaji this should be against RC-2021.3 - careful with this. If you have problem switching this PR, you should start over, ask @robyww . Thanks

**Pull request closed without merging**